README.md

```diff
@@ -21,6 +21,34 @@ CPython](https://github.com/python/cpython/blob/e0433c1e7/Doc/tools/rstlint.py).
 - focus on finding errors that are **not** visible to sphinx-build.
 
 
+## Using sphinx-lint
+
+To use sphinx-lint, simply run:
```

Comment:
Don't say simply:
* https://www.knowledgeowl.com/blog/posts/dont-say-simply-jim-fisher/
* https://www.youtube.com/watch?v=gsT2BBWBVmM&ab_channel=WritetheDocs


```suggestion
To use Sphinx Lint, run:
```

Reply:
Argh, you've pointed this out to me before, and I always forget! You're quite right!